background.js

```diff
@@ -60,35 +263,41 @@ MESSAGING FROM CONTENT SCRIPTS
 
 chrome.runtime.onMessage.addListener((msg, sender) => {
   
-  console.log('Background message received:');
-  
   // First, validate the message's structure.
   if ((msg.from === 'content') && (msg.subject === 'showPageAction')) {
     // Enable the page-action for the requesting tab.
     chrome.pageAction.show(sender.tab.id);
   }
   
+  //Clicking the Extension's Page Action
+  if ((msg.from === 'popup') && (msg.subject === 'pageActionClicked')) {
+    //Just in case we need it later...
+    console.log('Page action clicked.');
```

Comment:
Let's not console.log anything in production, as it pollutes the console for other kinds of debugging. You can use console.debug if you really need

Reply:
background.js and it's output and logs is not seen by the end user. it is not outputting that log to the frontend nor user's console, but rather to a hidden dev-only window that monitors the background ops